src/main/markdown/doc/latest/polymer-tutorial/create.md

```diff
@@ -33,12 +33,14 @@ For the **TodoList** project, we'll need to run `webAppCreator` with the followi
 
       _**Tip**: If you include the GWT SDK folder in your PATH environment variable, you won't have to specify the full path._
 
+      You may have to modify the `pom.xml` before you can run the application. Add `<type>pom</type>` to the gwt dependency, otherwise you will encounter an error. 
 
 2. Run the application in [SuperDevMode](articles/superdevmode.html).
 
      To check that the project was created correctly start the new app in SuperDevMode.
 
         $ cd TodoListApp
+        $ mvn war:exploded
```

Comment:
Including maven-war-plugin could be a good idea but for the webAppCreator instead. We don't want the user to modify so much the generated pom in this tutorial.
So, to me is safer to do `mvn war:exploded gwt:run` all in one line to avoid problems if he runs clean or he modifies the static stuff in webapp folder.